eclair-rpc/src/test/scala/org/bitcoins/eclair/rpc/EclairRpcClientTest.scala

```diff
@@ -32,89 +32,150 @@ class EclairRpcClientTest extends AsyncFlatSpec with BeforeAndAfterAll {
 
   val logger: Logger = BitcoinSLogger.logger
 
-  val bitcoindRpcClient: BitcoindRpcClient =
+  val bitcoindRpcClientF: Future[BitcoindRpcClient] =
     BitcoindRpcTestUtil.startedBitcoindRpcClient()
 
-  lazy val EclairNodes4(firstClient, secondClient, thirdClient, fourthClient) = {
-    val EclairNodes4(first, second, third, fourth) =
-      EclairRpcTestUtil.createNodeLink(bitcoindRpcClient)
-    clients ++= List(first, second, third, fourth)
-    EclairNodes4(first, second, third, fourth)
+  lazy val eclairNodesF: Future[EclairNodes4] = {
```

Comment:
is there any case where this is never called or can we speed things up by making this eager since its wrapped in a `Future` now